### ISSUE TITLE
Cannot browse minidlna DLNA share

### ISSUE THREAD

**Issue by Gert-dev** [phako/dleyna-server#6]
Since the update to 0.7.2 (or it could also be 0.7.1) in Arch, I cannot browse my minidlna server anymore using Rhythmbox or any other application indirectly using dleyna-server through grilo(-plugins).

I see the following in the logs when that happens:

```
dec 05 22:02:16 dleyna-server-service[15846]: Push host port: 0
dec 05 22:02:16 dleyna-server-service[15846]: [Logging settings]
dec 05 22:02:16 dleyna-server-service[15846]: Log Type : 0
dec 05 22:02:16 dleyna-server-service[15846]: Log Level: 0x13
dec 05 22:02:16 dleyna-server-service[15846]: [Network filtering settings]
dec 05 22:02:16 dleyna-server-service[15846]: Enabled : F
dec 05 22:02:16 dleyna-server-service[15846]: Entries: (null)
dec 05 22:02:17 dleyna-server-service[15846]: Entity: line 1: parser error : Start tag expected, '<' not found
dec 05 22:02:17 dleyna-server-service[15846]: status=ok
dec 05 22:02:17 dleyna-server-service[15846]: ^
dec 05 22:02:24 rhythmbox[15824]: [dleyna] ../grilo-plugins/src/dleyna/grl-dleyna-source.c:765: grl_dleyna_source_results error:GDBus.Error:com.intel.dleyna.OperationFailed: Browse operation failed: Invalid result
```

After that, nothing shows up in the results in Rhythmbox. This used to work fine, and I verified DLNA on the server still works through both an Android TV as well as VLC.

Thanks in advance

**Comment by phako** [comment]
Can you somehow check if it actually was 0.7.1 or 0.7.2?

**Comment by phako** [comment]
I can actually reproduce it 0.7.0. this is a tad weird. It's also not limited to minidlna, for Rygel it fails to even get the name of the server properly

**Comment by phako** [comment]
This is actually a GUPnP bug, introduced with 9a6534a445c122af938bd53251e8f64866f0abe6, but I am not sure why this isn't visible in the GUPnP tools -> Because I already ported them to the new API

**Comment by phako** [comment]
https://gitlab.gnome.org/GNOME/gupnp/-/issues/58

**Comment by phako** [comment]
Please retry with gupnp-1.4.1 once it hits arch.

**Comment by Gert-dev** [comment]
Confirmed fixed with 1.4.1 and by testing with Rhythmbox. Thanks for the super-fast response and resolution times!

Also nice to see that you added a test to prevent it from regressing again, too few projects do that.

As a good Linux citizen, I also flagged the gupnp package as out of date out of date in Arch 😉.

**Comment by phako** [comment]
yeah well, with proper test coverage this would not have happened in the first place :-/
